tools/ci_build/github/azure-pipelines/templates/orttraining-linux-gpu-ortmodule-test-ci-pipeline.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+REQDIR="requirements_torch1.11.0_cu11.3";
+
+if [ $5 == "Schedule" ]; then
```

Comment:
So you haven't run this pipeline, and nobody knows it works or not? If ever it had an issue, we need to wait another day to see if a fix works or not?

Reply:
I ran the nightly pipeline, and made sure it is successful before doing a PR or pushing new things to the PR.
this is the last run for the nightly pipeline : https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=700195&view=logs&j=52024a78-4e9d-503c-68de-2b0746163a14&t=7122c2d7-eecc-5787-8ed9-b9f92db23c12

You can change the build trigger and schedule it 2-3 minutes after your push to test it.